test/python/transpiler/test_unroller.py

```diff
@@ -98,3 +85,17 @@ def test_unroll_1q_chain_conditional(self):
         ref_circuit.u1(pi, qr[0]).c_if(cr, 1)
         ref_dag = circuit_to_dag(ref_circuit)
         self.assertEqual(unrolled_dag, ref_dag)
+
+    def test_no_gate_decomposition(self):
```

Comment:
I think in this test is better to just run the pass (as in the previous `test_unroll_no_basis`) to avoid retest the full transpiler code.

Reply:
That test tests that no-basis unrolls all the way to U, CX. But, that shouldn't happen since those aren't valid bases according to https://github.com/Qiskit/qiskit-terra/issues/1697#issuecomment-456909795. Are you suggesting I alter this test?

Reply:
Yes. You can test exactly that by just running the pass without running the transpiler.

Reply:
Ok. I'll wait for Jay's clarification on your other comment and then do that.

Reply:
Now that U and CX are a correct basis (see clarification above), I think this test must either be deleted or modified.
It can be modified by creating an instance of the qiskit.circuit.Gate class and setting gate.decompositions to None, then check that the Unroller raises an error.